Cargo.toml

```diff
@@ -133,6 +133,8 @@ toml = "0.5"
 tree-sitter = "0.20"
 tree-sitter-bash = { git = "https://github.com/tree-sitter/tree-sitter-bash", rev = "7331995b19b8f8aba2d5e26deb51d2195c18bc94" }
 tree-sitter-c = "0.20.1"
+tree-sitter-clojure = { git = "https://github.com/prcastro/tree-sitter-clojure", branch = "update-ts"}
+tree-sitter-cpp = { git = "https://github.com/tree-sitter/tree-sitter-cpp", rev="f44509141e7e483323d2ec178f2d2e6c0fc041c1" }
```

Comment:
```suggestion

```

Reply:
Sorry about that, happened when I was merging with the commits on master

Reply:
No worries ^.^